ui/StatusQ/src/StatusQ/Components/private/chart/Library/Library.js

```diff
@@ -0,0 +1,135 @@
+.pragma library
+
+.import "Polyfills.js" as Polyfills
+.import "Chart.bundle.js" as ChartJs
+.import "chartjs-plugin-crosshair.js" as CrosshairLib
+.import "chartjs-plugin-datalabels.js" as DataLabelsLib
+
+
+/*! 
+    /This file is used to provide the necessary wrappers for Chart.js to work in QML
+    /It is loaded after Chart.js and plugins and provides the necessary functions
+    /NOTE: Some plugins work out of the box, others need to be adapted to work in QML
+!*/
+
+(function(global){
+    
+    var color = global.Color
+    Chart.helpers.merge(Chart.defaults.global, {
+        // Default options
+        events: [
+            "mousemove",
+            "mouseout",
+            "click"
+        ]
+    })
+
+    // QML rendering plugin
+    Chart.plugins.register({
+        afterDraw: function(chart) {
+            chart.canvas.requestPaint()
+        }
+    })
+
+    var MOUSE_EVENTS = {
+        click: "clicked",
+        mousemove: "positionChanged",
+        mouseenter: "entered",
+        mouseout: "exited",
+        mousedown: "pressed",
+        mouseup: "released"
```

Comment:
Can we support `wheel` as well? I'm sure we'll have to use it, e.g. for zooming in and out

Reply:
I haven't seen any listener to handle wheel events, but we can add it later if needed. There's a zoom plugin that handles wheel events, but we're not using it. We can add it later if we'll need the plugin.

There's also the option of calling `doZoom` from crosshair plugin on wheel events. So we'll see whatever makes sense for our needs